src/network/serverpackethandler.cpp

```diff
@@ -716,6 +716,18 @@ void Server::handleCommand_ClientReady(NetworkPacket* pkt)
 			peer_id, major_ver, minor_ver, patch_ver,
 			full_ver);
 
+	std::vector<std::string> players = m_clients.getPlayerNames();
+	NetworkPacket list_pkt(TOCLIENT_UPDATE_PLAYER_LIST, 0, peer_id);
+	list_pkt << (u16) players.size();
+	for (const std::string &player: players) {
+		list_pkt << (u8) PLAYER_ADD_INIT <<  player;
```

Comment:
PLAYER_LIST_INIT, PLAYER_LIST_ADD, PLAYER_LIST_REMOVE are better name, this enum is globally available